docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -64,62 +107,41 @@ propagated to Alluxio.
   </property>
   <property>
     <name>alluxio.zookeeper.address</name>
-    <value>[zookeeper_hostname]:2181</value>
+    <value>zkHost1:2181,zkHost2:2181,zkHost3:2181</value>
   </property>
 </configuration>
 ```
 
-### Configure additional Alluxio properties
-
-Similar to above, add additional Alluxio properties to `core-site.xml` of Hadoop configuration in Hadoop directory on each node.
- For example, change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
-
-```xml
-<property>
-  <name>alluxio.user.file.writetype.default</name>
-  <value>CACHE_THROUGH</value>
-</property>
-```
-
-Alternatively, you can also append the conf path (i.e. `/<PATH_TO_ALLUXIO>/conf`) containing [`alluxio-site.properties`](Configuration-Settings.html) to Presto's JVM config at `etc/jvm.config` under Presto folder. The advantage of this approach is to have all the Alluxio properties set within the same file of `alluxio-site.properties`.
+#### Enable data locality
 
-```bash
-...
--Xbootclasspath/p:<path-to-alluxio-conf>
-```
-
-Also, it's recommended to increase `alluxio.user.network.netty.timeout` to a bigger value (e.g. `10min`) to avoid the timeout
- failure when reading large files from remote worker.
-
-### Enable `hive.force-local-scheduling`
+It is recommended to collocate Presto with Alluxio so that Presto workers can read data locally. An important option to enable in Presto is `hive.force-local-scheduling`, which forces splits to be
+scheduled on the same node as the Alluxio worker serving the split data. By default, `hive.force-local-scheduling` in Presto is set to `false`, and Presto will not attempt to schedule the work on the same machine as the Alluxio worker node.
```

Comment:
Can we discuss the tradeoff of setting `hive.force-local-scheduling` to true?

Reply:
I don't really see a reason setting this to `false` for Alluxio